### ISSUE TITLE
don't open it

### ISSUE THREAD

**Issue by pontiggia** [TOG6-6/discord-token-grabber#7]
The script already have a webhook url inside, so first edit the const WEBHOOK = ""

**Comment by ajeethbt** [comment]
Yeah noticed that the person should edit it and make it "INSERT WEBHOOK HERE"

**Comment by brooklxn** [comment]
cant wait to spam this webhook lol
